### DISCUSSION TITLE
MenuItem commands.

### DISCUSSION THREAD

**Discussion by CosminCAS123** [AvaloniaUI/Avalonia#16036] [Q&A] [answered]
Hey. I'm trying to put some commands on the menu-items of a Context Menu. Here for example 
	
```
<MenuItem Header="Remove"
		Command="{Binding RemoveTaskListCommand}">
					<MenuItem.Icon >
						<materialIcons:MaterialIcon Foreground="Red"
									Kind="TrashCanOutline"></materialIcons:MaterialIcon>
					</MenuItem.Icon>
</MenuItem>
```

It throws an error saying that it cannot resolve the property or method of name RemoveTaskListCommand. The DataContext is set properly. If I just make an empty button, assigning that command works perfectly fine. I've also tried it using the PointerPressed events and Avalonia Behaviours, but that doesn't work as well. Help?

**Comment by rabbitism** [comment]
I cannot provide any suggestion without your data context definition.

**Reply by CosminCAS123** [reply]
The data context is just a view-model. All other bindings work, just this one on the menu item doesn't

**Comment by CosminCAS123** [comment, marked as answer]
It worked by just adding x:DataType to MenuItem.  All bindings were appearing design-time , so I thought I wouldn't need it.
